connector/src/main/java/org/jboss/as/connector/deployers/ra/processors/ParsedRaDeploymentProcessor.java

```diff
@@ -150,7 +150,7 @@ public void deploy(DeploymentPhaseContext phaseContext) throws DeploymentUnitPro
                 });
 
             }
-            builder.addListener(new AbstractResourceAdapterDeploymentServiceListener(registration, deploymentUnit.getName(), deploymentResource, bootstrapCtx, deploymentUnit.getName()) {
+            builder.addListener(new AbstractResourceAdapterDeploymentServiceListener(registration, deploymentUnit.getName(), deploymentResource, bootstrapCtx, deploymentUnit.getName(), false) {
```

Comment:
Because it's possible to configure the parameter only for RA activated by resource-adapters subsystem in standalone/domain.xml Those are ones deployed "as is" in a rar and activated by an internal ironjacamar.xml file